flutter/realm_flutter/pubspec.yaml

```diff
@@ -39,7 +39,6 @@ dev_dependencies:
   build_cli: ^2.1.3
   json_serializable: ^6.1.0
   lints: ^1.0.1
-  test: ^1.20.1
 
 # TODO: Remove once flutter_test from sdk is updated. 
```

Comment:
You should remove the dep override as well..

Reply:
removed it